fsutil.py

```diff
@@ -1,5 +1,12 @@
 import logging
-import win32api
+try:
+    import win32api
+    use_windows_api = True
```

Comment:
This bothers me a little bit -- if the import fails due to "package management misadventure" (Windows user not installing win32api), then it's going to think "oh im not on windows do linux things" which I don't think is right (and then may get into a mess with macOS when it's time for that too). Part of this will be mitigated by eventually being smarter and having an install-able package that will get the dependencies right. The good news is that if we have these well-defined catboxes for OS-specific differences we should be pretty safe.

Reply:
Ah, good call. I will look for a different way to check the host OS rather than assume from the failed import.

Reply:
The newest commit checks the platform explicitly with:
```python
import platform
if platform.system() == "Windows":
    import win32api
    use_windows_api = True
elif platform.system() == "Linux":
    # Running on linux
    import os
    import getpass
    use_windows_api = False
```
What do you think of it this way?

Reply:
Love u. Yes, I like that better.